### ISSUE TITLE
FeatReq: Prompts mode column

### ISSUE THREAD

**Issue by Emasoft** [new-sankaku/stable-diffusion-webui-simple-manga-maker#11]
Please add a Prompts mode column to the Manga Maker. This is going to be a great tool, and to be used correctly it should allow the user to edit the prompts in place in each vignette frame. And even render them directly there. Prompt should also specify the LoRA to use for style, characters, settings, etc. (since the only way to have consistency between images in the manga is by using custom LoRA). LoRA should be also easily replaced, changed or upgraded in all pages or in a certain interval, so a find and replace tool should be added to edit /change all prompts in all pages at the same time. Ideally a switch should be added to the find and replace tool to choose between editing the scene description text or the LoRA section text at the end (or even the speech bubbles text).   
  
  
Example pic of the Manga Maker Prompt Mode column:  
  

![IMG_1323](https://github.com/new-sankaku/stable-diffusion-webui-simple-manga-maker/assets/713559/d03ecaa7-c999-4372-a31a-af2d80230d26)

**Comment by new-sankaku** [comment]
I think this is the right direction for evolution. It might be better to use Stable diffusion web ui, or start Forge in api mode to fire and run the API.

The problem is whether Adetailer and Upscaler can be used via API. This is because if high quality cannot be achieved, images may end up being created manually.

I plan to investigate.

**Comment by 3blackbar** [comment]
Hey man, how do i upload image on website demo? theres no button for that

**Comment by new-sankaku** [comment]
>[3blackbar](https://github.com/3blackbar)

#13 
thank you for contacting!
I've made it a separate issue. I'll take care of it.

**Comment by new-sankaku** [comment]
Added T2I settings to the layer panel.
Also, we have added Base settings.
Images can be generated with prompts for T2I settings and Base settings for each panel.
Added prompt replacement function to menu.
Supported by calling Stable diffusion WebUI API.
I couldn't support it because Forge doesn't have an API (I'll think about this in the future).
It has been implemented functionally, but its usability has not yet been evaluated.
It may be possible to get better results by using Image2Image of the rough sketch (future implementation).

Watch the video for quick details.

https://www.youtube.com/watch?v=-H4IYfimToo
